src/Randomizer.SMZ3.Tracking/Configuration/TrackerConfigProvider.cs

```diff
@@ -176,7 +176,7 @@ public virtual ICollection<string> GetAvailableProfiles()
             return Directory
                 .GetDirectories(_basePath)
                 .Select(x => (new DirectoryInfo(x)).Name)
-                .Where(x => x != "Templates")
+                .Where(x => x != "Templates" && !string.IsNullOrEmpty(x))
```

Comment:
Is this even possible?

Reply:
Ah, yeah. You're correct. Originally I wasn't exactly sure what caused the problem for Digi, so I was probably a bit too overzealous here. But now that I identified what the actual problem was, this can be removed since I don't think this is possible.